download/download.go

```diff
@@ -19,6 +20,7 @@ import (
 const federalRevenue = "https://www.gov.br/receitafederal/pt-br/assuntos/orientacao-tributaria/cadastros/consultas/dados-publicos-cnpj"
 const listOfCNAE = "https://cnae.ibge.gov.br/images/concla/documentacao/CNAE_Subclasses_2_3_Estrutura_Detalhada.xlsx"
 const retries = 10
+const regexHttp = `http(s?)//`
```

Comment:
Isso ainda faz que a expressão regular seja compilada a cada iteração. Que tal?

```suggestion

var httpRegex = regexp.MustCompile(`(http)(s?)(//)`)
```